app/mailers/rock_pebble_mailer.rb

```diff
@@ -0,0 +1,56 @@
+class RockPebbleMailer < ApplicationMailer
+  def confirmation_message(info)
+    address = info[:rock_email]
+    name = info[:pebble_name]
+    @message = "#{name} has requested you as a Rock through the Rock and Pebble
+                Program on Paired.tech. Please visit #{paired_link} to accept or deny
+                this request."
+    mail(to: address, subject: "Someone has requested you as a Rock!")
+  end
+
+  def rock_accepted_message(info)
+    address = info[:pebble_email]
+    name = info[:rock_name]
+    @message = "#{name} has accepted your Rock request. Please use their contact
+                information provided in your Paired Rock and Pebble dashboard
+                to get in touch with them and say Hi!"
+    mail(to: address, subject: "Your Rock request has been accepted!")
+  end
+
+  def rock_denied_message(info)
+    address = info[:pebble_email]
+    reason = info[:reason]
+    name = info[:rock_name]
+    @message = "#{name} declined your Rock request. The reason
+                given was: #{reason}. Even though this request did not work out
+                please go to #{paired_link} and request someone else as your Rock!"
+    mail(to: address, subject: "Your Rock request was not accepted.")
+  end
+
+  def pebble_relationship_discontinued(info)
+    address = info[:pebble_email]
+    reason = info[:reason]
+    name = info[:rock_name]
+    @message = "#{name} has decided to end your Rock and Pebble relationship.
+                The reason given is: #{reason}. Please feel free to follow up with
+                any feedback that you may have (always be specific, actionable and kind!)
+                or just to say 'Thanks' for the time given mentoring. You can go
```

Comment:
I'd suggest changing "the time given mentoring" to "the time they spent mentoring you" or "the time they shared with you."